quickjs-libc.c

```diff
@@ -3882,17 +3882,41 @@ static JSValue js_print(JSContext *ctx, JSValue this_val,
     const char *str;
     size_t len;
 
+#ifdef _WIN32
```

Comment:
There are barely a couple of lines of shared code here. Perhaps defining the whole function once for Unix one for Windows would be more readable here.

Reply:
Good call! Separated into two definitions, ready for another look